src/main/java/codeu/controller/RegisterServlet.java

```diff
@@ -0,0 +1,18 @@
+package codeu.controller;
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+/**
+* Servlet class responsible for user registration.
+*/
+public class RegisterServlet extends HttpServlet {

```

Comment:
It looks like the class doesn't do anything yet right? Want to add a comment that calls this out?